constants.py

```diff
@@ -27,7 +27,14 @@
 
 
 def parse_json_from_js(js_file):
-    """Extracts JSON object from JS file."""
+    """Extracts JSON object from JS file.
+
+    Args:
+        js_file: file. JS file containing JSON that needs to be parsed.
+
+    Returns:
+        dict. Parsed value of JSON in the JS file.
```

Comment:
`The dict representation of JSON object in the js file.`

Reply:
done